packages/cli/README.md

```diff
@@ -67,11 +64,14 @@ _If you're looking for programmatic usage, then refer to the underlying [@code-p
    };
    ```
 
-4. Define your custom categories.
+4. Optionally define your custom categories.
 
    ```js
    export default {
      // ...
+     plugins: [
+       // ...
+     ],
```

Comment:
The `// ...` allows us to omit the plugins for brevity:

```suggestion
     // ...
```

Reply:
Good point!